idb/postgres/internal/schema/setup_postgres.sql

```diff
@@ -41,9 +37,7 @@ CREATE TABLE IF NOT EXISTS asset (
   index bigint PRIMARY KEY,
   creator_addr bytea NOT NULL,
   params jsonb NOT NULL, -- data.basics.AssetParams; json string "null" iff asset is deleted
-  deleted bool NOT NULL, -- whether or not it is currently deleted
-  created_at bigint NOT NULL, -- round that the asset was created
-  closed_at bigint -- round that the asset was closed; cannot be recreated because the index is unique
```

Comment:
why removing these columns?

Reply:
Again,
`Indexer stores the round at which a particular row was inserted. I feel like this unwanted data and removing this would decrease the size of our indexer.`

Both created_at and closed_at store the round number.